ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml

```diff
@@ -60,7 +60,7 @@ tests:
       RELEASE_IMAGE_LATEST: release:latest
     env:
       BASE_DOMAIN: servicemesh.devcluster.openshift.com
-      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.15.6-multi
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.16.3-multi
```

Comment:
why not 4.16.5, which seems to be the latest?

Reply:
Done